src/features/community/components/BackButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import { OuterPanel } from "components/ui/Panel";
+import { Button } from "components/ui/Button";
+
+interface Props {
+  farmId: number;
+}
+
+export const BackButton: React.FC<Props> = ({ farmId }) => {
+  const goBackToFarm = () => {
+    console.log(farmId);
+    window.location.href = `${window.location.pathname}#/farm/${farmId}`;
```

Comment:
Check src\features\goblins\components\TownEntryModal.tsx its better to use that approach for back button. You dont need to pass farmId to that component and it seems faster to load, but that can be just me.

Reply:
Thanks for this, I never thought of using navigation for it, much cleaner and yes much faster than passing the prop ✌🏻😄